src/tactic/core.lean

```diff
@@ -130,15 +130,19 @@ each line break is decided independently -/
 meta def soft_break : format :=
 group line
 
+/-- format a list as a comma separated list, without any brackets. -/
```

Comment:
```suggestion
/-- Format a list as a comma separated list, without any brackets. -/
```
Do you think it makes sense to split the `format` defs around here to their own file, `meta.format`?